src/asm/lexer.c

```diff
@@ -1102,11 +1102,11 @@ static void discardComment(void)
 	lexerState->disableInterpolation = false;
 }
 
-/* Function to read a line continuation */
+/* Functions to read a line continuation or separator */
 
```

Comment:
A doc comment explaining what the function's return value means would be appreciated.